### PR TITLE
Release 0.36.0

### PR DESCRIPTION
**Pull request by bcardiff**
Preview at https://github.com/bcardiff/crystal/blob/chagelog/0.36.0/CHANGELOG.md

Note that this release will be available in [the new apt and rpm repositories post](https://crystal-lang.org/2020/08/24/announcing-new-apt-and-rpm-repositories.html).

If thinks looks good during the next couple of days, I would expect to do the releasing during next week.

Now... where is that test-ecosystem....

### CODE REVIEW

**Comment by asterite** [comment]
Does this mean we should stop merging PRs for a while?

**Comment by bcardiff** [comment]
Simple PRs are fine. Semantic & breaking changes should probably wait.

**Comment by straight-shoota** [comment]
FTR: Precursor is #9892 ([context](https://forum.crystal-lang.org/t/petition-to-release-crystal-0-36-soon/2860/15))

**Review by asterite — APPROVED** [review]
Looks great! 💯

**Comment by jwoertink** [comment]
Will this roll out with shards 0.12?

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,342 @@
...
+- Update distribution-scripts and use LLVM 10 on Linux packages. ([#9710](https://github.com/crystal-lang/crystal/pull/9710), thanks @bcardiff)
+- Update distribution-scripts and publish nightly packages to bintray. ([#9663](https://github.com/crystal-lang/crystal/pull/9663), thanks @bcardiff)
+- Initial Nix development environment. Use Nix for OSX CI instead of homebrew. ([#9727](https://github.com/crystal-lang/crystal/pull/9727), [#9776](https://github.com/crystal-lang/crystal/pull/9776), thanks @bcardiff)
+- Prevent recursive call in `bin/crystal` wrapper. ([#9505](https://github.com/crystal-lang/crystal/pull/9505), thanks @jhass)
+- Allow overriding `CRYSTAL_PATH` in `bin/crystal` wrapper . ([#9632](https://github.com/crystal-lang/crystal/pull/9632), thanks @bcardiff)
+- Makefile: support changing the current crystal via env var and argument. ([#9471](https://github.com/crystal-lang/crystal/pull/9471), thanks @bcardiff)
+- Windows CI: Avoid looking for globally installed libs. ([#9507](https://github.com/crystal-lang/crystal/pull/9507), thanks @oprypin)
+- Windows CI: Patch LLVM to fix broken build. ([#9907](https://github.com/crystal-lang/crystal/pull/9907), thanks @oprypin)
```
Comment (oprypin) [inline CHANGELOG.md:331]: Those two definitely qualify as just "CI improvements and housekeeping"

**Comment by bcardiff** [comment]
Yes @jwoertink. I believe is already shipped on nightlies that way.

**Comment by Daniel-Worrall** [comment]
the blogpost link is malformed

**Comment by asterite** [comment]
Here's a link: https://github.com/crystal-lang/crystal/blob/17d085d96db01fdbdb52e64aab393f43b3e96e55/CHANGELOG.md

**Comment by bcardiff** [comment]
I've updated the link in the OC to https://github.com/bcardiff/crystal/blob/chagelog/0.36.0/CHANGELOG.md (since I have typo in the branch name 😅 )

**Comment by Daniel-Worrall** [comment]
I meant the new apt and rpm repositories post :)

File: CHANGELOG.md [hunk 2]
```diff
@@ -260,12 +265,15 @@
...
 - **(breaking-change)** Disallow keywords as block argument name. ([#9704](https://github.com/crystal-lang/crystal/pull/9704), thanks @asterite)
 - **(breaking-change)** Fix `a-b -c` incorrectly parsed as `a - (b - c)`. ([#9652](https://github.com/crystal-lang/crystal/pull/9652), [#9884](https://github.com/crystal-lang/crystal/pull/9884), thanks @asterite, @bcardiff)
 - **(breaking-change)** Check abstract def implementations with splats, default values and keyword arguments. ([#9585](https://github.com/crystal-lang/crystal/pull/9585), thanks @waj)
 - **(breaking-change)** Make abstract def return type warning an error. ([#9810](https://github.com/crystal-lang/crystal/pull/9810), thanks @bcardiff)
 - **(breaking-change)** Error when abstract def implementation adds a type restriction. ([#9634](https://github.com/crystal-lang/crystal/pull/9634), thanks @waj)
-- **(breaking-change)** Don't interpret Proc typedefs as aliases. ([#10254](https://github.com/crystal-lang/crystal/pull/10254), thanks @HertzDevil)
+- **(breaking-change)** Don't interpret Proc typedefs as aliases. ([#10254](https://github.com/crystal-lang/
+crystal/pull/10254), thanks @HertzDevil)
```
Comment (Sija) [inline CHANGELOG.md:275]:
```suggestion
- **(breaking-change)** Don't interpret Proc typedefs as aliases. ([#10254](https://github.com/crystal-lang/crystal/pull/10254), thanks @HertzDevil)
```

**Review by straight-shoota — APPROVED** [review]

**Review by straight-shoota — APPROVED** [review]

**Review by asterite — APPROVED** [review]
It's happening! 🎉

**Pull request merged**